### PR TITLE
feat: panel title component

### PR DESCRIPTION
**Pull request by lja1018**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* add panel title component

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

File: apps/calendar/src/components/panelgrid/panelTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { cls } from '@src/util/cssHelper';
+import { Template } from '@src/components/template';
+import { TemplateName } from '@src/template/default';
+
+type PanelTitleTemplate = Extract<TemplateName, 'milestoneTitle' | 'taskTitle' | 'alldayTitle'>;
+
+interface Props {
+  width: number;
+  template: Extract<TemplateName, 'milestone' | 'task' | 'allday'>;
+  model: any;
```
Comment (jung-han) [inline apps/calendar/src/components/panelgrid/panelTitle.tsx:12]: model type 정의가 필요할 것 같아요~

**Review by jung-han — APPROVED** [review]
완료합니다~

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Review by jungeun-cho — APPROVED** [review]
이견없습니다~
화밋으로 말하였듯이 타이틀 템플릿 연동 부분은 이후에 수정작업이 있겠네요.

File: apps/calendar/src/components/panelgrid/panelTitle.tsx (hunk 1 above)
Comment (lja1018) [reply apps/calendar/src/components/panelgrid/panelTitle.tsx:12]: 템플릿 연동 작업과 같이 진행하겠습니다.

**Pull request merged**